Sources/SwiftGtk/enums/ArrowType.swift

```diff
@@ -0,0 +1,55 @@
+import CGtk
+
+/// Used to indicate the direction in which an arrow should point.
+///
+/// https://docs.gtk.org/gtk3/enum.ArrowType.html
+public enum ArrowType {
+    /// Represents an upward pointing arrow.
+    case up
+    
```

Comment:
Don't put empty lines between the cases, there's not anything wrong with doing that, but the rest of the project doesn't do that and it's best to be consistent where possible.